database/definition-variablen.txt

```diff
@@ -0,0 +1,84 @@
+Variable
+
+# Approved
+* "yes": this shall be put online, it is approved
+* "no": this one is confirmed to happen but misses some important information
+* "nada": this one is not going to happen this year
+* "": keep on checking this if there are news
+
+# Datestart: YYYYMMDD
+# Datenend: YYYYMMDD
+
+# Label: the name we use in the list
+# Name: The name we use in the detailed view
+# Shortname: no use yet, usable in social media etc
+# Hashtag: to be used in social media
+# Homepage: URL
+# City: English City name (could be changed one day to an international id, let's see ...)
+# Country: the two-digit country code
```

Comment:
There are different schemes for these codes. Perhaps https://en.wikipedia.org/wiki/ISO_3166-1_alpha-2 ?

Reply:
sounds reasonable, thanks "They are the most widely used of the country codes published by ISO (the others being alpha-3 and numeric), and are used most prominently for the Internet's country code top-level domains (with a few exceptions)"